### DISCUSSION TITLE
Metrics not working using WSL

### DISCUSSION THREAD

**Discussion by alexanderzeillinger** [vitodeploy/vito#176] [Q&A]
I am running VitoDeploy over WSL and have the "remote-monitor" service installed.

Metrics don't show anything though, should it work or is this a WSL issue?

![image](https://github.com/vitodeploy/vito/assets/126621/007bb4ac-4617-4b84-afdb-ad3181b389f2)

**Comment by saeedvaziry** [comment]
@alexanderzeillinger you need to make sure that you have a cronjob running the Laravel's scheduler on Vito.

https://laravel.com/docs/11.x/scheduling#running-the-scheduler

**Comment by alexanderzeillinger** [comment]
Thanks, that solved it - the cron service was not running.
